### ISSUE TITLE
ADA Compliance Problem with Headers

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/universalviewer#4]
#  Multiple empty `<h1>` tags
Documentation: https://webaim.org/techniques/semanticstructure/
## Current behavior
- tags are empty
- tags are called multiple times on a page

## Expected behavior
- Populate tag with title of object: `this.$title = $('<h1>' + this.object.title + '</h1>');`
- Only one `<h1>` tag in the page.
   (Either remove all other `<h1>` tags or place `<h1>` items in an if statement so only the one required tag is called when the page is loaded.)

**Comment by verbalhanglider** [comment]
After looking into this issue, I think I have a better understanding. The empty h1 tags appear to be related to separate functionality that can be triggered by what is in the manifest records. 

```html
<div class="placeMarkerDetails" style="display: none;">
11
<h1></h1>
<p>
</div>
```

```html
<div class="middle">
<div class="content">
11
<h1></h1>
<div>
<p class="message scroll">
</div>
</div>
<div class="buttons">
<a class="confirm btn btn-primary" href="#" target="_parent">
Confirm
</a>
1
<button type="button" class="btn btn-default close" tabindex="0">
Cancel
</button>
</div>
</div>
```

```html
<div class="middle">
<div class="content">
11
<h1></h1>
<div>
<p class="message scroll">
<div class="buttons">
<a class="cancel btn btn-primary" href="#" target="_parent">
Cancel
</a>
</div>
</div>
</div>
<div class="buttons">
1
<button type="button" class="btn btn-default close" tabindex="0">
Close
</button>
</div>
</div>
```

```html
<div class="content">
11
<h1></h1>
<div>
<p class="message scroll">
<div class="buttons">
<a class="logout btn btn-primary" href="#" target="_parent" style="display: none;">
Logout
</a>
<a class="login btn btn-primary" href="#" target="_parent">
Login
</a>
<a class="cancel btn btn-primary" href="#">
Cancel
</a>
</div>
</div>
</div>
```

I'd like to investigate further to figure out precisely what in the manifest should be populating these headers.
